src/librustdoc/clean/mod.rs

```diff
@@ -448,10 +415,60 @@ impl Clean<Item> for doctree::Module {
     }
 }
 
+pub trait Attributes {
+    fn has_word(&self, &str) -> bool;
+    fn value<'a>(&'a self, &str) -> Option<&'a str>;
+    fn list<'a>(&'a self, &str) -> Option<&'a [Attribute]>;
+    fn list_def<'a>(&'a self, &str) -> &'a [Attribute];
```

Comment:
Oh, this isn't needed at all. I thought the default [here](https://github.com/mitaa/rust/commit/2a28b69948e13ec09a8a7701fa4d9001e880ad5f#diff-05c3c8b7c08a25764218b8c18d8204b7L435) was something other than `&[]`